README.md

```diff
@@ -85,7 +85,11 @@ For information on how to install for use with the cloud environment, please see
 1. Fork our repo from https://github.com/publiclab/plots2.
 2. In the console, download a copy of your forked repo with `git clone https://github.com/your_username/plots2.git` where `your_username` is your GitHub username.
 3. Enter the new **plots2** directory with `cd plots2`.
-4. Install gems with `bundle install --without production mysql` from the rails root folder, to install the gems you'll need, excluding those needed only in production. You may need to first run `bundle update` if you have older gems in your environment from previous Rails work.
+4. Steps to install gems:
+    * You may need to first run `bundle update` if you have older gems in your environment from previous Rails work. If you get an error message like `Your Ruby version is 2.x.x, but your Gemfile specified 2.4.4` then you need to install the ruby version 2.4.4 using `rvm` or `rbenv`.
```

Comment:
I think this line should be removed.
> You may need to first run `bundle update` if you have older gems in your environment from previous Rails work.

 I would argue against running `bundle update`. 
When people clone Plots2 it comes with a `Gemfile.lock` file that has working versions of all the gems we need. Running `bundle update` will update gems in the project that may break or not work. `bundle install` is enough for this.

Reply:
@siaw23 
So if I will use `bundle install` here, then will I need `bundle install --without production mysql` in the next line.
Or should I just make it one point and remove `bundle update` at all?

Reply:
Replace `bundle update` with `bundle install`. That should be enough.

Reply:
I have made the changes.
@siaw23 Thank you so much :smile: